macleann/enchiridion-client
# Fix #3 and #4 to render playlists based on logging in

# Fix playlists rendering based on being logged in or not

Allows visitors or registered users to view all playlists whether they're logged in or not
Allows logged in users to view only their own playlists
Updates the name of the site in `index.html`
Updates the `NavBar.js` links
Supports those links in `ApplicationViews.js`
Adds a new provider wrapper in the `GodProvider.js`


Fixes #3 View all playlists and #4 View my playlists

## Type of change


# How to Test?

Make sure you've followed all installation and setup instructions in the README
**Also requires pulling the most recent `main` branch for the [enchiridion-server repo](https://github.com/macleann/enchiridion-server) as well**

```
git fetch origin nm-playlists
git checkout nm-playlists
npm start
```

- [ ] Go to `http://localhost:3000/playlists`
- [ ] You should see the initial seeded playlist
- [ ] Log in with username `mooglyg` and password `password`
- [ ] Go to `http://localhost:3000/playlists/my-playlists`
- [ ] You should see the initial seeded playlist
- [ ] Log out
- [ ] Log in as a different user or make a new account
- [ ] Go to `http://localhost:3000/playlists/my-playlists`
- [ ] The page should say "No playlists found."

# Checklist:
